### ISSUE TITLE
abstractTree: recurseNodeToExtractTree should work off the root

### ISSUE THREAD

**Issue by pkra** [krautzource/aria-tree-walker#33]
recurseNodeToExtractTree searches for data-owns-id below the current node. It should look inside the root (to avoid failure in aria-owns-like situations).

**Comment by pkra** [comment]
> recurseNodeToExtractTree searches for data-owns-id below the current node. It should look inside the root (to avoid failure in aria-owns-like situations).

I think the source of the problem might be different. We do search widely, but too widely - across the entire document

https://github.com/krautzource/aria-tree-walker/blob/fc35c85c1302df4256ac45af43000b3149404db8/lib/abstractTree.js#L32

We should search the treebase instead.

**Comment by pkra** [comment]
This is a really dumb bug and due to #25 since we used to have proper/unique IDs.
